frontend-project/src/pages/letters/list/index.tsx

```diff
@@ -0,0 +1,54 @@
+import { PageHeaderWrapper } from '@ant-design/pro-layout';
+import ProTable, { ProColumns, ActionType } from '@ant-design/pro-table';
+import React, { useRef } from 'react';
+import { formatMessage } from 'umi-plugin-react/locale';
+
+import { Letter, fetchLettersPage } from '@/services/letters';
+
+const TableList: React.FC<{}> = () => {
```

Comment:
IMO `FC` ładniej wygląda po named imports (skoro i tak ich używamy)
```ts
import React, { useRef, FC } from 'react';
```
i tutaj po prostu FC
bo importowanie useRef po named import a FC po bodajże synthetic imports to trochę inconsistency